osd/master_node_restarted.json

```diff
@@ -3,6 +3,6 @@
     "service_name": "SREManualAction",
     "cluster_uuid": "${CLUSTER_UUID}",
     "summary": "Master node restarted",
-    "description": "Your cluster was unstable due to a master node being stopped. We have restarted the master node and the cluster now appears healthy. Please avoid stopping your node instances in the future.",
+    "description": "Red Hat SRE detected that one of the master nodes was manually stopped. In order to restore the full resiliency feature of our control plane we have restarted that node. Please make sure to never manually stop any management infrastructure (master or infra nodes) as it will decrease the resiliency of your cluster and breach our support model.",
```

Comment:
"description": "Red Hat SRE has detected that one of the master nodes was manually stopped. In order to restore the full resiliency of our control plane we have restarted that node. Please make sure to never manually stop any management infrastructure (master or infra nodes) as it will decrease the resiliency of the cluster and potentially breach the contractual SLA. If you need help with that please file a support ticket."

to align with our team meeting discussion

Reply:
I don't think a customer can breach an SLA (we provide the SLA), that's not the wording here.

>  "… decrease the resiliency of the cluster and prevent Red Hat from delivering on the contractual SLA."

Wording to that effect – things Cu did prevents us from delivering the contractual agreement. An English native fluent in legalize could probably word this better.

Reply:
I like your proposed wording
go for it